docs/quickstart.md

```diff
@@ -18,6 +18,8 @@ The template will deploy in your Azure subscription the Following resources:
 - [IoT Hub](https://azure.microsoft.com/en-us/services/iot-hub/)
 - [Azure Function](https://azure.microsoft.com/en-us/services/functions/)
 - [Redis Cache](https://azure.microsoft.com/en-us/services/cache/)
+- [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) (when opted in to use Azure Monitor)
```

Comment:
afaik this one is always deployed in the existing template (needed by the function)

Reply:
Ok I saw your changes let's discuss

Reply:
This is related to another discussion we had - I reverted to always deploy Application Insights for the Function monitoring. Resolved this with c29006667d50cd427ab3ba4de00e010df6bf7341